sunpy/net/cdaweb/cdaweb.py

```diff
@@ -94,17 +94,21 @@ def _do_search(self, query):
                                            query['end_time']))
 
         if 'FileDescription' not in response:
-            raise RuntimeError
-
-        stimes = [f['StartTime'] for f in response['FileDescription']]
-        etimes = [f['EndTime'] for f in response['FileDescription']]
-        urls = [f['Name'] for f in response['FileDescription']]
-
-        return astropy.table.QTable(
-            {'Dataset': [query['dataset']] * len(stimes),
-             'Start time': Time.strptime(stimes, '%Y-%m-%dT%H:%M:%S.%fZ').iso,
-             'End time': Time.strptime(etimes, '%Y-%m-%dT%H:%M:%S.%fZ').iso,
-             'URL': urls})
+            # No results
+            return astropy.table.QTable(
```

Comment:
This shouldn't be returning a QTable at all it should be returning a QueryResponseTable (which is a subclass of QTable).

Reply:
This is an internal helper method, isn't it fine that this returns whatever as long as `search` returns the `QueryResponseTable` (which it does above)?

Reply:
I think if we are worried about this, we can follow up in a new PR but it works and I don't see the issue with it being a Qtable.